package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "react-apps",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
testing libraries can be added in `devDependencies` or you can even remove them as you will be not using them for now.